.travis.yml

```diff
@@ -1,6 +1,7 @@
 language: ruby
+before_install:
+  - gem install bundler
```

Comment:
I'm wondering why did you add this line? Travis comes with bundler. However this line actually fixes my build errors for Ruby 2.1.5. Funny thing is that Travis is using Bundler 1.7.6 with Ruby 2.1.5 and fails to install the gem (should use local source). But on my dev env it works fine, same setup, Ruby 2.1.5 + bundler 1.7.6.